src/test/scala/carldata/series/TimeSeriesTest.scala

```diff
@@ -389,5 +389,29 @@ class TimeSeriesTest extends FlatSpec with Matchers {
     series.removeOutliers(1, 10) shouldBe expected
   }
 
+  it should "check if 2 ts is almost equal (true)" in {
```

Comment:
It would be nice to add one more test here.
Where series have exactly the same values, but on of the series is longer.

In this case amolstEqual should be false

Reply:
Like this:
```
    val vs = Vector(1f, 2.2f, 2.4f, 10.7f)
    val vs2 = Vector(1f, 2.2f, 2.4f, 10.707f)
    val series1 = TimeSeries(idx1, vs)
    val series2 = TimeSeries(idx2, vs2)

    TimeSeries.almostEqual(series1,series2, 0.0001f) shouldBe false
```
?